### DISCUSSION TITLE
Visualization of .case, .geo, .00001, ... files with Trame (web app)

### DISCUSSION THREAD

**Discussion by DieuMerciKIMPOLO** [Kitware/trame#220] [Trame/VTK]
Good morning
I have data from a simulation, in several files. The extensions of the different files are:
```
RESULTS.case
results.currentnb.00001
results.dissip.00001
results.efforts.00001
results.fouriernb.00001
results.geo
results.input_thermal_flux.00001
results.localtime.00001
results.pressure.00001
results.scalar1.00001
results.tempc.00001
results.total_pressure.00001
results.turbener.00001
results.turbvisc.00001
results.velocity.00001
results.wall_temperature.00001
results.yplus.00001

```
I would like to visualize them in a web application either with Trame or vtk.js. I don't know the proper components to do this. Can you help me in this direction?
If there is an example, I'm a taker.

Thank you.

**Comment by jourdain** [comment]
It seems to be an Ensight mesh which should be able to read using [vtkEnSightGoldReader](https://vtk.org/doc/nightly/html/classvtkEnSightGoldReader.html). Then you can look at [vtk-example website](https://kitware.github.io/vtk-examples/site/) to learn how to show some mesh with vtk (python). Then you can use trame to make it display and interact with a web page.
